JavaScript/Default.sublime-keymap

```diff
@@ -51,4 +51,24 @@
             { "key": "selection_empty", "operator": "equal", "operand": false, "match_all": true }
         ]
     },
+
+    // Add indented line in square brackets
```

Comment:
I've been wondering if this is something we'd want to have universally much like the same binding for curlies. I *do* have it in my user key bindings, though I also believe that having the closing bracket at the end of the line instead of in a new line might be more common than for curlies.

Now that I think about it, I should probably add the same for normal parentheses. :thinking:

Reply:
I never prefered closing brackets at the same line, personally.

Well, may it be a useful global behavior? I don't know. Maybe rather unlikely to happen and hard to judge.

Once, we have this binding in each package, we can still make a move :)

Reply:
In the meanwhile it seems more safe to treat this syntax-specific. 

_These bindings have been copied from JSON._